symfony.lock

```diff
@@ -227,6 +227,9 @@
     "lcobucci/jwt": {
         "version": "3.2.2"
     },
+    "league/csv": {
+        "version": "9.6.1"
+    },
```

Comment:
Cannot comment on an empty file as it seems. The below script 'wait-for-it.sh' seems redundant.

Reply:
Yeap. File from other task. Sorry for that and thx